src/rendering/renderers/shared/texture/__tests__/TexturePool.test.ts

```
@@ -0,0 +1,383 @@
+            {
+                expect(tex.source.autoGenerateMipmaps).toBe(true);
+            });
+
+            // All normal textures should NOT have mipmaps
+            normalTextures.forEach((tex) =>
+            {
+                expect(tex.source.autoGenerateMipmaps).toBe(false);
+            });
+
+            // Return all to pool
+            mipmapTextures.forEach((tex) => pool.returnTexture(tex));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)